wpilibc/src/main/native/include/frc/smartdashboard/SendableBase.h

```diff
@@ -16,7 +16,7 @@
 
 namespace frc {
 
-class SendableBase : public Sendable {
+class SendableBase : public virtual Sendable {
```

Comment:
Is virtual inheritance required here?

Reply:
No, just forgot to undo it. Reverted the file.